README.md

```diff
@@ -135,11 +135,10 @@ If you are in the development process and using the UI, you may need to change y
 
 
  1. Retrieve the full source code from this repository: `git clone https://github.com/bioinformatics-ua/dicoogle.git`
- 2. Navigate to the web app directory: "./dicoogle/src/main/resources/webapp/"
- 3. Build the web application: `npm install`
-    - The full instructions can be read [here](dicoogle/src/main/resources/webapp/README.md).
- 4. Navigate to the project's base directory, and build the parent Maven project by calling `mvn install`.
- 5. The resulting jar file can be found in "./dicoogle/target".
+ 2. Navigate to the project's base directory, and build the parent Maven project by calling `mvn install`.
+    - Note: if you want, you can skip the npm part: `mvn install -Dskip.npm`

```

Comment:
I think it's not clear that this part refers to building the webapp. Maybe we should either extend this note or add a second list of steps to do it manually.